lib/libvcc/generate.py

```diff
@@ -654,6 +653,7 @@ def tbl40(a, b):
 	const struct vpi_ref	*ref;
 
 	int			nsrc;
+	unsigned		nsub;
 	const char		**srcname;
```

Comment:
If VCC also produced a `const char *subname[nsub]`, could that improve any error messages ?

Reply:
see above.
having the name of all the (custom) subs available would also require to modify the VRT_CTX in the preamble of the generated C functions. Or we would need to function-wrap the C code for VCL `call`.
In my mind, this is overkill.

Reply:
and I forgot to mention that `VRT_CTX` is const.